src/services/google-auth.js

```diff
@@ -29,6 +30,10 @@ export const googleLogin = async () => {
       server.defaults.headers["Authorization"] = "Bearer " + response.data.access;
       localStorage.setItem("access_token", response.data.access);
       localStorage.setItem("refresh_token", response.data.refresh);
+
+      // TODO: Remove once backend changes done
+      const jwtDecoded = jwt_decode(response.data.access);
+      localStorage.setItem("userId", jwtDecoded.user_id);
```

Comment:
This can be removed after pulling the changes in the backend and can be replaced with `localStorage.setItem("userId", response.data.id)`